jekyll/_ccie/preview.md

```diff
@@ -242,8 +230,7 @@ No changes, the same process as for the CCIE 1.0 <https://circleci.com/docs/ente
          ]
      }
      ```
- * vm-service configuration:
-   * AWS User need to have this permissions. It might be the same user as for S3, but then it needs to have both set of permissions.
+6. Configure the vm-service. AWS User need to have this permissions. It might be the same user as for S3, if so, it needs to have both sets of permissions.
```

Comment:
I think my Grammarly extension would want that to read `The AWS user needs to have these` but this is a nitpick :)